.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Verify
+Can you see anything in the logs? Can you debug why this happens? **Send a pull request**! Bug fixes and documentation fixes are very welcome.
```

Comment:
I like encouraging people to submit pull requests if they can, but we want to make sure that they're not scared off if they don't feel able/willing to do that much work.